game-headed/src/main/java/org/triplea/game/client/ui/javafx/screens/MainMenuControls.java

```diff
@@ -0,0 +1,91 @@
+package org.triplea.game.client.ui.javafx.screens;
+
+import java.util.function.Function;
+
+import javax.swing.SwingUtilities;
+
+import org.triplea.game.client.ui.javafx.screen.ControlledScreen;
+import org.triplea.game.client.ui.javafx.screen.RootActionPane;
+import org.triplea.game.client.ui.javafx.screen.ScreenController;
+import org.triplea.game.client.ui.javafx.util.FxmlManager;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import games.strategy.engine.framework.map.download.DownloadMapsWindow;
+import javafx.beans.binding.Bindings;
+import javafx.beans.binding.NumberBinding;
+import javafx.fxml.FXML;
+import javafx.scene.Node;
+import javafx.scene.layout.VBox;
+import lombok.NoArgsConstructor;
+
+/**
+ * Controller class for the main screen navigation menu.
+ */
+@NoArgsConstructor
+public class MainMenuControls implements ControlledScreen<ScreenController<FxmlManager>> {
+
+  @FXML
+  private VBox mainOptions;
+
+  private ScreenController<FxmlManager> screenController;
+
+  @VisibleForTesting
+  MainMenuControls(final VBox mainOptions) {
+    this.mainOptions = mainOptions;
+  }
+
+  @FXML
+  @VisibleForTesting
+  void initialize() {
+    mainOptions.lookupAll(".button").forEach(node -> {
+      final Function<Node, NumberBinding> hoverBinding = n -> Bindings.when(n.hoverProperty()).then(-10).otherwise(0);
```

Comment:
Could it make sense to turn it into a private static method?

Reply:
Yeah, this needs to be updated at some point.
I haven't really bothered to change the implementation, I just wanted to change the underlying system